met-api/src/met_api/services/slug_generation_service.py

```diff
@@ -0,0 +1,16 @@
+"""Service for generating slugs."""
+from slugify import slugify
+
+
+class SlugGenerationService:
+    """Service for generating slugs."""
+
+    @staticmethod
+    def generate_slug(text: str) -> str:
+        """Generate a slug from the provided text."""
+        normalized_text = slugify(
+            text,
+            to_lower=True,  # NOSONAR # to_lower is a valid paramter for awesome-slugify
+            max_length=100
+        )
+        return normalized_text
```

Comment:
write some texts for this function..
just write a fixture with expected and return value
like Broadway Subway Project to broadway-subway etc
Also max length can be a configuration item.. 
default it to 20 or something

Reply:
Added a test with fixtures and made the max length a config item. Not very confident about defaulting to 20, I feel it's too short. I kept it 100 unless 20 is a strong feeling